tests/Features/Helpers.php

```diff
@@ -38,8 +38,9 @@ function mockUser()
 });
 
 it('throws error if method do not exist', function () {
-    test()->name();
-})->throws(\ReflectionException::class, 'Call to undefined method PHPUnit\Framework\TestCase::name()');
+    $user = new User();
```

Comment:
Why you had to change this test?

Reply:
It seems that phpunit doesn't throw a ReflectionException anymore

Reply:
```php
it('throws error if method do not exist', function () {
    test()->name();
})->throws(\ReflectionException::class, 'Call to undefined method PHPUnit\Framework\TestCase::name()');
```

this will output:

```
There were 2 failures:

1) P\Tests\Features\Helpers::it_throws_error_if_method_do_not_exist
Failed asserting that exception of type "ReflectionException" is thrown.

/data/projects/open-source/pest/src/Kernel.php:57

2) P\Tests\Features\Helpers::it_throws_error_if_method_do_not_exist
Failed asserting that exception of type "ReflectionException" is thrown.
```

Reply:
Well, that's because the method "name()" now exists. Just rename it something else.

Reply:
lol, you're right :facepalm: , fixed